sdks/python/apache_beam/yaml/yaml_provider.py

```diff
@@ -876,8 +877,10 @@ def _parse_window_spec(spec):
       return beam.WindowInto(window_fn)
 
   @staticmethod
+  @beam.ptransform_fn
+  @maybe_with_exception_handling_transform_fn
```

Comment:
Do we expect this to every have errors (that we'd want to ignore)?

Reply:
The only case for Python I can think of is if it is somehow not JSON parseable (Possibly a custom provider upstream that does not return Row)

For Java, I could see there being other issues (for starters, doesn't ERROR throw and exception when it logs?)

Regardless, this change was mostly for making it easier to hot-swap LogForTesting in example pipelines that define error_handling Write transforms. But I could also create a custom Log transform within the examples test file if you think that makes more sense?

Reply:
Sure, that's fine.